checkpoint_ngfw/help.md

```diff
@@ -32,6 +33,46 @@ The connection configuration accepts the following parameters:
 
 ### Actions
 
+#### Install Policy
+
+This action is used to install a policy to selected targets.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|discard_other_sessions|boolean|True|True|Discard all other user sessions. This can fix errors when objects are locked by other sessions|None|
+|install_on_all_cluster_members_or_fail|boolean|False|True|Relevant for the gateway clusters. If true, the policy is installed on all the cluster members. If the installation on a cluster member fails, don't install on that cluster|None|
+|policy_package|string|standard|True|Policy package to install e.g. "standard"|None|
+|targets|[]string|['target name']|True|On what targets to execute this command. Targets may be identified by their name, or object unique identifier. e.g. ["checkpoint_fw"]|None|
+
+Example input:
+
+```
+"input": {
```

Comment:
Input key should not be here, it's the contents of that key. Same way we don't show `output` key in example outputs because the user does not see that in the product. This would confuse them.